src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
if you do this, this would also include adding the non-inverse classes back to the `TaskViews` which are not inverse (of which there are many)